capstone/src/main/resources/templates/savedorganizations.html

```diff
@@ -4,12 +4,16 @@
     <meta>
     <title>Saved Organizations</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
     <script src="user.js"></script>
```

Comment:
I am curious -- does every page need to load every javascript file?

Reply:
Not every page, but most of the pages do.